tpdatasrc/tpgamefiles/scr/tpModifiers/brutal_throw.py

```diff
@@ -6,9 +6,20 @@
 
 print "Registering Brutal Throw"
 
+#Check if the weapon can only be thrown
+def IsThrownOnly(wpnType):
```

Comment:
Have you found an equivalent function in the engine? Or is it likewise hardcoded?

Reply:
As far as I can tell there is no function like this in the engine.  In fact the one place I can think of where it should be used on the C++ side is done incorrectly (same bug as my feat).  I will add a version in C++, fix the bug and make the C++ version accessible in python.

Reply:
Wait, so is Javelin actually throw only in the game, or just by the rules? (btw technically you could use it for melee, but with a penalty)

Reply:
The javelin is thrown only in game.  I can't see any way to make a melee attack with it.